docs/src/solvers/IntegralSolvers.md

```diff
@@ -2,15 +2,15 @@
 
 The following algorithms are available:
 
-  - `QuadGKJL`: Uses QuadGK.jl. Requires `nout=1` and `batch=0`.
+  - `QuadGKJL`: Uses QuadGK.jl. Requires `nout=1` and `batch=0`, in-place is not allowed.
   - `HCubatureJL`: Uses HCubature.jl. Requires `batch=0`.
-  - `VEGAS`: Uses MonteCarloIntegration.jl. Requires `nout=1`.
+  - `VEGAS`: Uses MonteCarloIntegration.jl. Requires `nout=1`. Works only for `>1`-dimensional integrations.
   - `CubatureJLh`: h-Cubature from Cubature.jl. Requires `using IntegralsCubature`.
   - `CubatureJLp`: p-Cubature from Cubature.jl. Requires `using IntegralsCubature`.
-  - `CubaVegas`: Vegas from Cuba.jl. Requires `using IntegralsCuba`.
+  - `CubaVegas`: Vegas from Cuba.jl. Requires `using IntegralsCuba`, `nout=1`. 
```

Comment:
Is this actually true? I'm not sure it is. All of the Cuba methods allow larger nout?

Reply:
Sure, `Cuba` methods (including `vegas`) supports vector-valued integrands, but since our tests did specifically limit the checks for `nout=1`, I didn't want to check in the docs as of now. Basically I just synced the docs with what is tested.

Also I did try to run quickly `Integrals.CubaVegas` for vector-valued integrands but I got a few different errors with a few different integrands, but didn't have time to investigate it thoroughly. I can report back here when I find time.